benchmark/single-source/MapReduce.swift

```diff
@@ -133,7 +133,7 @@ public func run_MapReduceShortString(_ N: Int) {
 
 @inline(never)
 public func run_MapReduceClass(_ N: Int) {
-  let numbers = (0..<1000).map { NSDecimalNumber(value: $0) }
+  let numbers = (0..<1000).map { NSDecimalNumber(integerLiteral: $0) }
```

Comment:
@airspeedswift ```NSDecimalNumber(integerLiteral: $0)``` can be replaced with ```Decimal($0)``` if we have a way to convert ```Decimal``` to ```Int```.  I don't think ```Int(NSDecimalNumber(decimal: $0))``` makes sense. Any suggestions?